M2/Macaulay2/m2/chaincomplexes.m2

```diff
@@ -137,15 +137,15 @@ net ChainComplex := C -> (
 	  a := s#0;
 	  b := s#-1;
 	  horizontalJoin between(" <-- ", apply(a .. b,i ->
-		  stack (net moduleAbbrv(C_i, C_i)," ",net i)))))
```

Comment:
This file is moved in the Complexes PR, so you might as well rebase on top of that PR I think.